server/adaptors/integrations/__data__/repository/aws_elb/data/sample.json

```diff
@@ -0,0 +1,106 @@
+[
+    {
+        "timestamp": "2023-06-21T21:49:46.564177Z",
+        "aws":{
+            "elb":{
+                "client":{
+                    "ip":"15.248.1.132",
+                    "port":33556
+                },
+                "target_ip":"172.31.32.179",
+                "target_port": 80,
+                "request_processing_time": 0.005,
+                "target_processing_time": 0.001,
+                "response_processing_time": 0.000,
+                "elb_status_code":200,
+                "target_status_code":200,
+                "received_bytes": 521,
+                "sent_bytes":271,
+                "http":{
+                    "port":80,
+                    "version":"1.1"
+                },
+                "matched_rule_priority": "0",
+                "request_creation_time": "2023-06-21T21:49:46.563000Z"
+            }
+        },
+        "traceId": "Root=1-6493707a-062e3d077d0e7f5759a83c88",
```

Comment:
where is the traceId originating from ?

Reply:
When I parsed it I mapped it to the AWS ELB access log Trace_Id field: https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html